docs/writing-stories/index.md

```diff
@@ -4,7 +4,7 @@ title: 'How to write stories'
 
 <YouTubeCallout id="P0WHt_L0-2g" title="How to write Storybook stories | Component Story Format" />
 
-A story captures the rendered state of a UI component. It’s a function that returns a component’s state given a set of arguments.
+A story captures the rendered state of a UI component. It’s an object with annotations and a render function that returns a component’s state given a set of arguments.
```

Comment:
@nkabrown we have room for improvement here and we can go with something along these lines:
```suggestion
A story captures the rendered state of a UI component. It's an object with annotations that describe the component's behavior and appearance given a set
of arguments.
```